src/Components/Hero/index.tsx

```diff
@@ -10,14 +10,15 @@ export default class Hero extends React.Component {
             <div className="hero">
                 <div className="hero--info">
                     <h1>Python Pizza</h1>
-                    <h2>
+                    <h2>August 2020 - TBA</h2>
```

Comment:
We should maybe consider adding "Berlin" somewhere, either here on the same with the date, or in the H1 above ("Python Pizza Berlin" for example). What do you think?